tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -158,6 +158,25 @@ Verify "Notebook Images Are Building" Is Not Shown When No Images Are Building
     Wait Until All Builds Are Complete  namespace=redhat-ods-applications
     RHODS Notification Drawer Should Not Contain  message=Notebooks images are building
 
+Verify Notifications Appears When Notebook Builds Finish And Atleast One Failed
+    [Documentation]    Verifies that Notifications are shown when Notebook Builds are finished and atleast one fails
+    [Tags]    Sanity
+    ...       ODS-470  ODS-718
+    Clear Dashboard Notifications
+    ${build_name}=  Search Last Build  namespace=redhat-ods-applications    build_name_includes=pytorch
+    Delete Build    namespace=redhat-ods-applications    build_name=${build_name}
+    ${new_buildname}=  Start New Build    namespace=redhat-ods-applications    buildconfig=s2i-pytorch-gpu-cuda-11.4.2-notebook
+    Wait Until Build Status Is    namespace=redhat-ods-applications    build_name=${new_buildname}   expected_status=Running
+    ${failed_build_name}=  Provoke Image Build Failure    namespace=redhat-ods-applications
+    ...    build_name_includes=tensorflow    build_config_name=s2i-tensorflow-gpu-cuda-11.4.2-notebook
+    ...    container_to_kill=sti-build
+    Wait Until Build Status Is    namespace=redhat-ods-applications    build_name=${newbuild_name}     expected_status=Complete
+    RHODS Notification Drawer Should Contain  message=builds completed successfully
```

Comment:
could you try looking for the message "5 out 6 builds completed successfully"? replacing 5 and 6 with the real numbers. 
Example here https://issues.redhat.com/browse/RHODS-1649

Reply:
updated

Reply:
could you please test it nce again after the latest changes?

Reply:
Tested